libr/util/logging.c

```diff
@@ -0,0 +1,127 @@
+/* radare - LGPL - Copyright 2007-2013 - pancake */
+
+#define LOGGING_CONFIGSTR_SIZE 512
```

Comment:
why set a limit for this? just use the heap

Reply:
I agree with this. Only thing I could think of is that in this way we could use logging even when the memory is finished (if we care about this).

Reply:
Hurd dislikes fixed size buffers for paths, but yeah i assume there must be a sane limit. even more for logging